utils.js

```diff
@@ -12,23 +12,29 @@
 function getRoomName () { // eslint-disable-line no-unused-vars
```

Comment:
@guusdk, Is the function used at all? I see it has a `eslint-disable-line no-unused-vars` so we should probably check whether it's still used and, it not, we should remove it.